### DISCUSSION TITLE
I can't connect to the codespace

### DISCUSSION THREAD

**Discussion by SinaSohrab** [community/community#110982] [Codespaces] [answered]

**Comment by Akash1134** [comment]
Hi @SinaSohrab , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by SinaSohrab** [reply, marked as answer]
Thank you, my problem is solved. One of my VS Code plugins was disabled

**Reply by Akash1134** [reply]
Thanks for the update. Glad you figured it out 🙂
